install-on-arch.sh

```diff
@@ -90,7 +89,8 @@ then
 	(cd ~/.srcs/$HELPER/ && makepkg -si )
```

Comment:
I think the package installation part is missing here. Maybe the line `$HELPER -S picom-jonaburg-git acpi candy-icons-git wmctrl alacritty playerctl dunst xmonad-contrib jq xclip maim rofi-greenclip` should be outside the `if` statement?

Reply:
Yeah I don't see a reason to keep it in the if statement if we are installing the helper automatically. I'll move it out.